relay-general/src/store/normalize/contexts.rs

```diff
@@ -177,10 +177,33 @@ fn normalize_os_context(os: &mut OsContext) {
     }
 }
 
+fn normalize_response(response: &mut ResponseContext) {
+    let headers = match response.headers.value_mut() {
+        Some(headers) => headers,
+        None => return,
+    };
+
+    if response.cookies.value().is_some() {
+        headers.remove("Cookie");
```

Comment:
Since `Cookie` is not a response header, I think we should use `Set-Cookie` here.

@marandaneto was the intention of the `cookies` field to contain the parsed contents of the `Set-Cookie` header?

Reply:
Not really, I just noticed that they were not scrubbed and opened the issue, but they were response headers and I was looking into the request headers, so it made sense.
Your call if it should be removed or not due to PII concerns.

Reply:
@jjbayer  pointed that `Cookie` is the request header and is not expected in the response.  We can just follow pattern and do what was proposed above, remove `set-cookie` header and convert those values to `cookies` when needed.